src/i18n/zh-cn/ui.ts

```diff
@@ -1,47 +1,64 @@
-import { UIDictionary } from '../translation-checkers';
-
-export default UIDictionary({
-    'a11y.skipLink': '跳转到内容',
-	'navbar.a11yTitle': '导航顶部',
-    // Site settings
+export default {
+	'a11y.skipLink': '跳转至内容',
+	'navbar.a11yTitle': '顶部',
+	// Site settings
 	'site.title': 'Astro 文档',
-	'site.description': '用更少的客户端 JavaScript 构建更快的站点。',
+	'site.description': '使用更少的客户端 Javascript 构建更快的网站。',
 	'site.og.imageSrc': '/default-og-image.png?v=1',
-	'site.og.imageAlt': '星辰浩瀚的太空中的 astro 标志，右侧前景中漂浮着一颗紫色的土星状行星。',
+	'site.og.imageAlt': '漫漫星空中的 Astro 图标，右侧前景中漂浮着紫色土星一般的星球',
 	// Left Sidebar
-	'leftSidebar.a11yTitle': '站点导航',
+	'leftSidebar.a11yTitle': '主要内容',
 	'leftSidebar.learnTab': '学习',
 	'leftSidebar.referenceTab': '参考',
-	'leftSidebar.noTranslations': '没有找到翻译内容',
-	'leftSidebar.viewInEnglish': '查看英文版本',
+	'leftSidebar.noTranslations': '未找到翻译',
+	'leftSidebar.viewInEnglish': '查看英文版文档',
 	// Right Sidebar
-	'rightSidebar.a11yTitle': '目录',
-	'rightSidebar.onThisPage': '在本页内',
-	'rightSidebar.overview': '概述',
-	'rightSidebar.more': '更多',
-	'rightSidebar.editPage': '编辑本页内容',
-	'rightSidebar.translatePage': '翻译本页内容',
-	'rightSidebar.joinCommunity': '加入社群',
+	'rightSidebar.a11yTitle': '次要内容',
+	'rightSidebar.onThisPage': '此页上',
+	'rightSidebar.overview': '概览',
+	'rightSidebar.community': '社群',
+	'rightSidebar.joinDiscord': '加入我们的 Discord 社群',
+	'rightSidebar.readBlog': '阅读博客',
+	'rightSidebar.openCollective': '参与众筹',
+	'rightSidebar.contribute': '贡献',
+	'rightSidebar.editPage': '编辑此页',
+	'rightSidebar.translatePage': '翻译此页',
+	'rightSidebar.github': 'GitHub 的 Astro 文档',
 	// `<ThemeToggleButton>` acessibility labels
-	'themeToggle.useLight': '使用浅色主题',
-	'themeToggle.useDark': '使用深色主题',
-	'rightSidebar.github': 'GitHub 上的 Astro 文档',
+	'themeToggle.useLight': '使用亮色主题',
+	'themeToggle.useDark': '使用暗色主题',
```

Comment:
This is entirely a difference caused by personal preference, and there is nothing wrong with the original translation.